section3/exercises/hashes.rb

```diff
@@ -1,31 +1,32 @@
 # In the below exercises, write code that achieves
 # the desired result. To check your work, run this
-# file by entering the following command in your terminal: 
+# file by entering the following command in your terminal:
 # `ruby section3/exercises/hashes.rb`
 
 # Example: Write code that prints a hash holding grocery store inventory:
 foods = {apples: 23, grapes: 507, eggs: 48}
 p foods
 
 # Write code that prints a hash holding zoo animal inventory:
-zoo = #YOUR CODE HERE
+zoo = {zebras: 12, monkeys: 34, emus: 56}
 p zoo
 
-# Write code that prints all of the 'keys' of the zoo variable 
+# Write code that prints all of the 'keys' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.keys
 
-# Write code that prints all of the 'values' of the zoo variable 
+# Write code that prints all of the 'values' of the zoo variable
 # you created above:
-# YOUR CODE HERE
+puts zoo.values
 
-# Write code that prints the value of the first animal of the zoo variable 
+# Write code that prints the value of the first animal of the zoo variable
 # you created above:
-# YOUR CODE HERE
+p zoo.keys[0]
```

Comment:
This prints the key `zebras` rather than the _value_ of the first animal of the zoo variable. I think what you're looking for here is: `p zoo[zoo.keys[0]]`